alacritty/src/renderer/mod.rs

```diff
@@ -643,6 +639,17 @@ impl QuadRenderer {
             );
             gl::EnableVertexAttribArray(4);
             gl::VertexAttribDivisor(4, 1);
+            // coloredGlyph attribute
```

Comment:
This comment's text should match the variable name and be terminated with a period.